src/rust/engine/process_execution/src/lib.rs

```diff
@@ -470,6 +484,14 @@ impl From<ProcessResultMetadata> for ExecutedActionMetadata {
   }
 }
 
+#[derive(Clone, Copy, Debug, Eq, PartialEq)]
+pub enum ProcessResultSource {
+  RanLocally,
```

Comment:
Naming: Maybe `ExecutedLocally` and `ExecutedRemotely` to better fit with the use of the word "executed" in other types (e.g., ExecutedActionMetadata)?

Reply:
Eh, yea... maybe. We've mostly isolation the "Execution" name to our REAPI usage, with the end-user name being `Process` (i.e., our python `Process` type used to be called `ExecuteRequest`, but no longer.)